README.md

```diff
@@ -1,5 +1,7 @@
 # Red Hat Developer Hub Installer for AI Software Templates
 
+> We are only supporting **GitHub Authentication**, **GitLab Authentication** is planned for future versions.
```

Comment:
I'm wondering if this should be marked as important same as what @johnmcollier did on the template repo here https://github.com/redhat-ai-dev/ai-lab-template/commit/4ec1c75ff3cb0cd1827bb454b8a4155a534092f9?

Thoughts?

Reply:
yeah sure that makes sense

Reply:
addressed in https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/20/commits/e41515d7a51f655a8160c0143eb35e0b273d007a